api/routes/users/put.endpoint.js

```diff
@@ -42,7 +42,7 @@ describe('users endpoint | PUT /users/:userID', async () => {
       expect(body).toMatchSnapshot();
     });
 
-    describe('when updating a valid user ID', async () => {
```

Comment:
![](https://media.giphy.com/media/l4pTgKSq2LDffuUIo/giphy-downsized-large.gif)

Reply:
It's like the PR equivalent of Groundhog Day.